### PR TITLE
Fix variable collision with imported package name

### PR DESCRIPTION
**Pull request by szepeviktor**
- [x] The contributed code is licensed under GPL v2.0
- [x] Contributor Licence Agreement (CLA) is signed
- [ ] util/update-modules has been ran
- [ ] Documentation updated
- [ ] Test suite update

Variable and package name were both "user".

### CODE REVIEW

File: src/go/lib/config/config_test.go [hunk 1]
```diff
@@ -129,14 +129,14 @@ func TestOverrideConfig(t *testing.T) {
 }
 
 func TestDefaultFiles(t *testing.T) {
-	user, _ := user.Current()
+	currentUser, _ := user.Current()
```
Comment (svetasmirnova) [inline src/go/lib/config/config_test.go]:
```suggestion
	current, _ := user.Current()
```

File: src/go/lib/config/config_test.go [hunk 2]
```diff
@@ -129,14 +129,14 @@ func TestOverrideConfig(t *testing.T) {
...
 	toolname := "pt-testing"
 
 	want := []string{
 		"/etc/percona-toolkit/percona-toolkit.conf",
 		fmt.Sprintf("/etc/percona-toolkit/%s.conf", toolname),
-		fmt.Sprintf("%s/.percona-toolkit.conf", user.HomeDir),
-		fmt.Sprintf("%s/.%s.conf", user.HomeDir, toolname),
+		fmt.Sprintf("%s/.percona-toolkit.conf", currentUser.HomeDir),
```
Comment (svetasmirnova) [inline src/go/lib/config/config_test.go]:
```suggestion
		fmt.Sprintf("%s/.percona-toolkit.conf", current.HomeDir),
```

File: src/go/lib/config/config_test.go [hunk 3]
```diff
@@ -129,14 +129,14 @@ func TestOverrideConfig(t *testing.T) {
...
+		fmt.Sprintf("%s/.%s.conf", currentUser.HomeDir, toolname),
```
Comment (svetasmirnova) [inline src/go/lib/config/config_test.go]:
```suggestion
		fmt.Sprintf("%s/.%s.conf", current.HomeDir, toolname),
```

**Review by svetasmirnova — CHANGES_REQUESTED** [review]
According to Go style guide local variables should have short names (<= 7 characters). Please either approve my suggestions or find better short name.

**Comment by szepeviktor** [comment]
Done 🍏 

These are my findings in variable naming
https://github.com/szepeviktor/uninformative-words

**Review by svetasmirnova — APPROVED** [review]

**Comment by artemgavrilov** [comment]
> Done 🍏
> 
> These are my findings in variable naming https://github.com/szepeviktor/uninformative-words

This guide conflicts with go conventions. Short names are ok when scope is small.

File: src/go/lib/config/config_test.go [hunk 4]
```diff
@@ -129,14 +129,14 @@ func TestOverrideConfig(t *testing.T) {
 }
 
 func TestDefaultFiles(t *testing.T) {
-	user, _ := user.Current()
+	current, _ := user.Current()
```
Comment (artemgavrilov) [inline src/go/lib/config/config_test.go:132]: As for me it can be just `u`

**Review by artemgavrilov — APPROVED** [review]

**Review by egegunes — APPROVED** [review]

File: src/go/lib/config/config_test.go (hunk 4 above)
Comment (svetasmirnova) [reply src/go/lib/config/config_test.go:132]: I think this is not really important, could be any short name.

**Pull request merged**